docking.js

```diff
@@ -1375,6 +1376,47 @@ const DockedDash = new Lang.Class({
             else
                 return false;
         }
+    },
+
+    /**
+     * Isolate overview to open new windows for inactive apps
+     */
```

Comment:
This extension of the Overview is simply to open a new window of a given App, when this App is already active in a different workspace. Example of the problem without this extension:

   - open gedit, then move to another workspace
   - open the overview and type "gedit"
   - hit enter to open the app
   - you will be shown the previous gedit window (in the other workspace).